### PR TITLE
DIP 25: Temporary Anonymous Zone community hub

### PR DESCRIPTION
**Pull request by catsnackattack**

### CODE REVIEW

File: DIPs/DIP-25.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+DIP: 25
+Title: Community Hub - Temporary Anonymous Zone
+Status: Accepted
+Themes: Community Involvement
+Tags: Event Production
```
Comment (franzihei) [inline DIPs/DIP-25.md:6]:
```suggestion
Tags: Community Hub, Event Production
```

File: DIPs/DIP-25.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Authors: Althea <althea@ethereum.org>
+Resources Required: Physical space, signage, seating, projector if possible
```
Comment (franzihei) [inline DIPs/DIP-25.md:8]: Projector is not possible, all hubs will be equipped with one TV-sized screen though (see Community Hub production guide)

**Review by franzihei — COMMENTED** [review]
looks great! I added the ``Communit Hub``tag which is helpful when browsing through https://devcon.org/dips/. :)

File: DIPs/DIP-25.md (hunk 2 above)
Comment (catsnackattack) [reply DIPs/DIP-25.md:8]: Oh sorry, knew that but forgot to update. Thank you!

**Review by franzihei — DISMISSED** [review]

File: DIPs/DIP-25.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+DIP: 25
```
Comment (ligi) [inline DIPs/DIP-25.md]:
```suggestion
DIP: 27
```

**Review by ligi — APPROVED** [review]

**Pull request merged**